idea/src/main/resources/schema/paddle-schema.json

```diff
@@ -173,29 +173,38 @@
     },
     "requirements": {
       "type": "object",
-      "properties": {
-        "file": {
-          "type": "string"
+      "anyOf": [
+        {
+          "properties": {
```

Comment:
Let's just drop the path to Requirements file for now. I suggest later to introduce copy-paste handler for Requirements file that would automatically convert Requirements format to Paddle format.